addons/website_event_exhibitor/models/event_sponsor.py

```diff
@@ -242,3 +243,30 @@ def _message_get_suggested_recipients(self):
                 reason=_('Sponsor')
             )
         return recipients
+
+    @api.model
+    def _search_get_detail(self, website, order, options):
+        domain = []
+        if options.get('event'):
+            event_id = self.env['ir.http']._unslug(options['event'])[1]
+            domain.append([('event_id', '=', event_id)])
+
+        fetch_fields = ['name', 'website_url', 'partner_name']
+        mapping = {
+            'name': {'name': 'name', 'type': 'text', 'match': True},
+            'website_url': {'name': 'website_url', 'type': 'text', 'truncate': False},
+            'partner_name': {'name': 'partner_name', 'type': 'text', 'match': True, 'html': True},
```

Comment:
`html: True`? :thinking:

Reply:
actually partner_name is not needed any more :smiley: